php/commands/search-replace.php

```diff
@@ -328,20 +341,51 @@ private function php_handle_col( $col, $primary_keys, $table, $old, $new ) {
 		return $count;
 	}
 
-	private function write_sql_row_fields( $table, $row_fields ) {
+	private function write_sql_row_fields( $table, $rows ) {
 		global $wpdb;
-		$sql = "INSERT INTO `$table` (";
-		$sql .= join( ', ', array_map(
-		function ( $field ) {
-			return "`$field`";
-		},
-		array_keys( $row_fields )
+
+		if(empty($rows)) {
+			return;
+		}
+
+		$insert = "INSERT INTO `$table` (";
+		$insert .= join( ', ', array_map(
+			function ( $field ) {
+				return "`$field`";
+			},
+			array_keys( $rows[0] )
 		) );
-		$sql .= ') VALUES (';
-		$sql .= join( ', ', array_fill( 0, count( $row_fields ), '%s' ) );
-		$sql .= ");\n";
-		$sql = $wpdb->prepare( $sql, array_values( $row_fields ) );
-		fwrite( $this->export_handle, $sql );
+		$insert .= ') VALUES ';
+		$insert .= "\n";
+
+		$sql = $insert;
+		$values = array();
+
+		$index = 1;
+		$count = count( $rows );
+		$export_insert_size = $this->export_insert_size;
+
+		foreach($rows as $row_fields) {
```

Comment:
```
if( ( $index % $export_insert_size == 0 && $index > 0 ) || $index == $count ) {
```

Meaning this is a complex conditional and difficult to review / understand the logic of. Can it be written more simply?